frontend/packages/core/src/NPS/emojiRatings.tsx

```diff
@@ -27,7 +32,13 @@ const getKey = (map, val): string => Object.keys(map).find(key => map[key] === v
  * @param setRating function which will return the given selected rating
  * @returns rendered EmojiRatings component
  */
-const EmojiRatings = ({ ratings = [], setRating }: EmojiRatingsProps) => {
+const EmojiRatings = ({
+  ratings = [],
+  setRating,
+  placement = "top",
+  size = "small",
```

Comment:
nit, can we rename to `iconSize` since we have `emojiSize`?

Reply:
Yeah thats a good call, adjust